src/components/pages/post-meetup/fields/tags-field/index.tsx

```diff
@@ -19,13 +25,14 @@ export const MeetupTagsField = () => {
           />
         }
         placeholder='입력 후 Enter'
-        required
         type='text'
+        value={field.state.value}
+        onChange={(e) => field.handleChange(e.target.value)}
       />
       <ul className='mt-0.5 flex flex-wrap px-2'>
         <li className='bg-mint-100 text-mint-700 flex-center gap-0.5 rounded-full px-2 py-0.5'>
           <p className='text-text-xs-medium'>#태그</p>
-          <Icon id='small-x' width={10} height={10} />
+          <Icon id='small-x' width={7} height={7} />
```

Comment:
아실지 모르겟는데 width, height 말고 className 지정으로도 가능합니다~
```suggestion
          <Icon id='small-x' className='size-1.75' />
```